cypress/e2e/auth.cy.ts

```diff
@@ -20,4 +43,19 @@ describe("Authentication tests", () => {
 
         cy.url().should("include", "/login");
```

Comment:
Is it worth checking that `cy.visit("/clients")` actually redirects to clients page before clicking the sign out button? 
(in case it stays on the login page despite `cy.visit("/clients")`)